src/diffusers/pipelines/shap_e/pipeline_shap_e.py

```diff
@@ -284,8 +284,8 @@ def __call__(
             frame_size (`int`, *optional*, default to 64):
                 the width and height of each image frame of the generated 3d output
             output_type (`str`, *optional*, defaults to `"pt"`):
-                The output format of the generate image. Choose between: `"np"` (`np.array`) or `"pt"`
-                (`torch.Tensor`).
+                The output format of the generate image. Choose between: `"pil"` (`PIL.Image.Image`), `"np"`
+                (`np.array`),`"latent"` (`torch.Tensor`), mesh (`MeshDecoderOutput`).
```

Comment:
```suggestion
                (`np.array`),`"latent"` (`torch.Tensor`), mesh ([`MeshDecoderOutput`]).
```

So that `MeshDecoderOutput` properly renders.